qutebrowser/extensions/loader.py

```diff
@@ -180,7 +227,15 @@ def _on_config_changed(changed_name: str) -> None:
                 cfilter = config.change_filter(option)
                 cfilter.validate()
                 if cfilter.check_match(changed_name):
-                    hook()
+                    try:
```

Comment:
This doesn't catch exceptions in the `if option is None:` part above - I guess the whole block should be in the `try:` instead?